pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/index/reader/ForwardIndexReader.java

```diff
@@ -916,4 +920,72 @@ default byte[][] getBytesMV(int docId, T context) {
   default int getNumValuesMV(int docId, T context) {
     throw new UnsupportedOperationException();
   }
+
+  // Functions for recording absolute buffer byte ranges accessed while reading a given docId
+
+  /**
+   * Returns whether the forward index supports recording the byte ranges accessed while reading a given docId
+   */
+  default boolean isBufferByteRangeInfoSupported() {
```

Comment:
Should we consider adding a method to return the underlying `PinotDataBuffer`?

Reply:
Sorry don't follow, why do we need that?